cmd/server/config/config.go

```diff
@@ -21,22 +22,19 @@ func DefaultConfig() *Config {
 			ProcessPaths: []string{},
 			ProxyPorts:   []int{},
 		},
-		Upgrade: UpgradeConfig{
-			Mode:         "watching",
-			WatchingPath: defaults.UpgradeWatchingPath,
-		},
+		AgentStrictTLSMode: false,
 	}
 }
 
 type Config struct {
-	Debug       bool                `yaml:"debug" json:"debug"`
-	Listen      string              `yaml:"listen" json:"listen"`
-	Proxy       string              `yaml:"proxy" json:"proxy"`
-	WhiteList   WhiteListConfig     `yaml:"white_list" json:"white_list"`
-	Upgrade     UpgradeConfig       `yaml:"upgrade" json:"upgrade"`
-	SystemAgent *config.AgentConfig `yaml:"systemagent" json:"systemagent"`
-	CSIProxy    *csiproxy.Config    `yaml:"csi-proxy" json:"csi-proxy"`
-	TLSConfig   *wintls.Config      `yaml:"tls-config" json:"tls-config"`
+	Debug              bool                `yaml:"debug" json:"debug"`
+	Listen             string              `yaml:"listen" json:"listen"`
+	Proxy              string              `yaml:"proxy" json:"proxy"`
+	WhiteList          WhiteListConfig     `yaml:"white_list" json:"white_list"`
+	SystemAgent        *config.AgentConfig `yaml:"systemagent" json:"systemagent"`
```

Comment:
to keep consistence with others:
```suggestion
	SystemAgent        *config.AgentConfig `yaml:"systemagent" json:"systemagent,omitempty"`
```